cpu/stm32_common/include/candev_stm32_params.h

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     candev_stm32
+ * @{
+ *
+ * @file
+ * @brief       STM32 CAN controller driver (bxCAN) default parameters
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @}
+ */
+
+#ifndef CANDEV_STM32_PARAMS_H
+#define CANDEV_STM32_PARAMS_H
+
+#include "can/device.h"
+#include "candev_stm32.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** Default STM32 CAN devices config */
+static const candev_stm32_conf_t candev_stm32_conf[] = {
+    {
+#if defined(CPU_FAM_STM32F0)
```

Comment:
this should be `#if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F1)` eg. on the `bluepill`its A11 and A12 for CAN, too